.eslintrc.js

```diff
@@ -2,6 +2,7 @@ const clientESLintConfig = require('./config/eslint');
 
 module.exports = Object.assign({}, clientESLintConfig, {
   env: Object.assign({}, clientESLintConfig.env, {
-    node: true
+    node: true,
+    jest: true,
```

Comment:
I don't think it's reasonable to add the env to every test file. Instead, is it possible to apply some eslint rules based on a filename pattern? A subfolder .eslintrc won't work here either unfortunately.